special-interest-groups/observability-tag/charter.md

```diff
@@ -0,0 +1,48 @@
+# OpenSearch Observability Technical Advisory Group (TAG) Governance Charter
+
+### Purpose
+
+The OpenSearch Project proposes to create a Observability Interest Group that focuses on observability using OpenSearch. This includes log analytics, traces, metrics using OpenSearch and related projects such as Prometheus, OpenTelemetry. This group will also establish guidelines for members, set architecture guidelines, and come up with recommendations which will then be reviewed by TSC for decision making. The group will provide feedback to the TSC before the final decision is made through voting of TSC members.
+
+### Responsibilities
+
+The following is a non-exhaustive sample list of activities and deliverables that are in-scope for this SIG
+
+* Drive feature development and enhancements for observability applications using OpenSearch (e.g. ingestion, plugins, Alerting, Anomaly Detection, Dashboards).
```

Comment:
Normally a TAG/SIG isn't building features but about building community and adoption. If this is for feature dev I don't think the scope should be this broad below.

Reply:
I agree. "drive feature development" should be rephrased.